pages/product.js

```diff
@@ -8,32 +8,47 @@ module.exports = {
   sizeListSpoiler: {xpath: '//label[contains(text(),"Size")]/following-sibling::div'},
   chooseSize: {xpath: '//label[contains(text(),"Size")]/following-sibling::div/ul/li[2]'},
   addToCartButton: {xpath: '//button[@id="button-cart"]'},
-  baseProductPrice: {xpath: '//span[@class="price-new"]'},
+  baseProductPrice: {xpath: '//div[@class="price"]/span'},
 
+  async checkColourExists() {
+    return await tryTo(() => I.seeElement(this.colourListSpoiler));
+  },
   
+  async chooseProductColour() {
+    let colourPrice = 0;
+    if (await this.checkColourExists()) {
+      I.click(this.colourListSpoiler);
+      I.click(this.chooseColour);
+      colourPrice = parseFloat((await I.grabTextFrom(this.chooseColour)).match('([0-9]*[.])[0-9]*')[0]);
```

Comment:
check for `replaceAll(/[^0-9\.]/g, "")` method for string. It will remove everything, but digits and dots. Current regex will not work for prices with comma `1,000.00`, which is highly expected.